### PR TITLE
fisher: update page to comply with Fisher 4.0

### PR DESCRIPTION
**Pull request by mbluemer**
Update the fisher page to comply with Fisher 4.0.

- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Thanks, @mbluemer!

Shame that the "show available plugins" example has gone away now though. Is there an equivalent? If not, no worries :-)

**Comment by mbluemer** [comment]
@sbrl Unfortunately it doesn't seem like there's an equivalent. The [v3 release notes](https://github.com/jorgebucaran/fisher/releases/tag/3.0.0) mention it's removal due to there no longer being a "preferred organization to look for packages" which I'm assuming means there may have been a central GitHub Org that was used for search.

And np, thanks for helping maintain this amazing package! It's such an integrated part of my daily routine now, happy to help in any way I can.

**Comment by sbrl** [comment]
Ah, no worries then. Let's merge as-is (as soon as we get a 2nd approval of course).

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**
